README.md

```diff
@@ -149,46 +73,63 @@ A real example with a client/app id and tenant id:
+| Default Cluster | (Options) If no cluster is selected when making a query, the default cluster will be used. |
 
-   ![Data Source Type](https://raw.githubusercontent.com/grafana/azure-data-explorer-datasource/main/src/img/config_1_select_type.png)
+### Additional settings
 
-3. In the name field, a default name is filled in automatically but it can be changed to anything.
+Additional settings are optional settings that can be configured for more control over your data source. Additional settings can be accessed by expanding the additional settings section at the bottom of the data source configuration page.
 
-4. You need 3 pieces of information from the Azure portal (see link above for detailed instructions):
+| Field  | Description                                 |
+| ------ | ------------------------------------------- |
+| Query timeout| This value controls the client query timeout.| 
+| Use dynamic caching | By enabling this feature Grafana will dynamically apply cache settings on a per-query basis, and the default cache max age will be ignored. The bin size for time series queries will be used to widen the time range and as cache max age. |
+| Cache max age | By default, the cache is disabled. If you want to enable the query caching please specify a max timespan for the cache to live. |
+| Data consistency | Query consistency controls how queries and updates are synchronized. Defaults to Strong. For more information, refer to [Query consistency](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/concepts/queryconsistency) | 
+| Default editor mode | This setting dictates which mode the editor will open in. Defaults to Visual. |
+| Default database | The default database will be used if no database is selected. A default cluster is required to select a default database. To load default database options, you must save the data source with a valid Azure connection. |
+| Use managed schema | If enabled, tables, functions, and materialized views are mapped to user-friendly names. |
+| Send username header to host | With this feature enabled, Grafana will pass the logged in user's username in the `x-ms-user-id` header and in the `x-ms-client-request-id` header when sending requests to ADX. It can be useful when tracking needs to be done in ADX. |
 
-   - **Tenant Id** (Azure Active Directory -> Properties -> Directory ID)
-   - **Client Id** (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
-   - **Client Secret** ( Azure Active Directory -> App Registrations -> Choose your app -> Keys)
+### Configuring On-Behalf-Of authentication (Beta)
 
-5. Paste these three items into the fields in the Azure Data Explorer API Details section:
-   ![Azure Data Explorer API Details](https://raw.githubusercontent.com/grafana/azure-data-explorer-datasource/main/src/img/config_2_azure_data_explorer_api_details.png)
+_This feature is in Beta and subject to breaking changes_
```

Comment:
```suggestion
{{%/* admonition type="caution" */%}}
This feature is in Beta and subject to breaking changes
{{%/* /admonition */%}}
```

Reply:
just a heads up that if you're building your branch locally, the admonition shortcode I added here might not render for you, but it should render live when you publish/merge

Reply:
oooo this is really cool!